src/plugins/query_enhancements/public/query_assist/components/query_assist_bar.tsx

```diff
@@ -106,6 +108,32 @@ export const QueryAssistBar: React.FC<QueryAssistInputProps> = (props) => {
 
   if (props.dependencies.isCollapsed) return null;
 
+  const { dependencies } = props;
+
+  const datasetSupported = PPL_SUPPORT_DATASET_TYPES.includes(
+    dependencies.query.dataset?.type || ''
```

Comment:
Can we use `selectedDataset` if it's the same as `dependencies.query.dataset`?

Reply:
Sure, updated.